### PR TITLE
Ballot paper size config

### PR DESCRIPTION
**Pull request by beausmith**
I was looking for a way to implement this using classes which doesn't seem possible for setting the `@page` settings.

PagedJS does have a [feature for named pages](https://www.pagedmedia.org/about-paged-media/pagedjs-cheatsheet/) however this does not seem to be supported for the first page.

https://user-images.githubusercontent.com/28444/107608510-0b8ccf80-6bf1-11eb-9853-1692a7da20e3.mov

### CODE REVIEW

File: apps/election-manager/src/utils/ballotLayout.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { BallotPaperSize, Election } from '@votingworks/types'
+
+export const getBallotLayoutPageSize = (election: Election): string =>
+  (election.ballotLayout?.paperSize || BallotPaperSize.Letter).toLowerCase()
+
+export default {
+  getBallotLayoutPageSize,
+}
```
Comment (eventualbuddha) [inline apps/election-manager/src/utils/ballotLayout.ts:8]: Is this useful?

**Review by eventualbuddha — APPROVED** [review]
🎉

File: apps/election-manager/src/data/defaultElection.json [hunk 2]
```diff
@@ -17,6 +17,9 @@
       "state": "Estado de Muestra"
     }
   },
+  "ballotLayout": {
```
Comment (carolinemodic) [inline apps/election-manager/src/data/defaultElection.json:20]: should we add to the sample elections in /fixtures as well?

Comment (beausmith) [reply apps/election-manager/src/data/defaultElection.json:20]: Yes, definitely. I was getting tired and wanted to push this MVP. 😴

File: apps/election-manager/src/utils/ballotLayout.ts (hunk 1 above)
Comment (beausmith) [reply apps/election-manager/src/utils/ballotLayout.ts:8]: Not necessary. Thought some default export was necessary for linter happiness. Will remove. 👍

**Pull request merged**